site/includes/homepage.css

```diff
@@ -23,7 +25,7 @@ CSS for the module browse page
 }
 
 body{
-  font-family: Verdana,Geneva,sans-serif;
+  font-family: 'Raleway',sans-serif;
```

Comment:
The reason the main body doesn't use Raleway as its font is that numbers do not display well in this font. If we go this route we will need to add some sort of number span to display digits.

Reply:
Adding `font-variant-numeric: lining-nums;` "correctly" aligns the numbers with the rest of the text. The mixed fonts just looked a bit off to me.